tests/integration/suite_test.go

```diff
@@ -32,15 +37,15 @@ var _ = BeforeSuite(func(ctx SpecContext) {
 	testRunId := randstr.String(6)
 	tempDir := GinkgoT().TempDir()
 	githubClient := testconfig.NewGitHubClient()
-	gitAuth := git.UrlTokenAuthMethod(testconfig.Cfg.GitHubToken)
+	GitAuth = git.UrlTokenAuthMethod(testconfig.Cfg.GitHubToken)
```

Comment:
local vars start with a lower case

Reply:
ah, sorry, forgot to revert this change, removed